typescript/web/src/components/import-button/import-images-modal/modal-dropzone/modal-dropzone.tsx

```diff
@@ -212,11 +233,18 @@ export const ImportImagesModalDropzone = ({
                 });
               }
 
-              const form = new FormData();
-              form.append("image", acceptedFile.file);
+              // SUPABASE upload
```

Comment:
Can this comment be removed?

Reply:
This is the only instance of the practical asymmetry between Supabase and S3 storage, so I'd prefer to keep it until we definitely discard Supabase for storage.